apps/mudita-center-e2e/wdio.conf.ts

```diff
@@ -84,6 +85,7 @@ export const config: Options.Testrunner = {
       toRelativePath(TestFilesPaths.privacyPolicyTest),
       toRelativePath(TestFilesPaths.licenseTest),
       toRelativePath(TestFilesPaths.helpWindowCheckOfflineTest),
+      toRelativePath(TestFilesPaths.mcHomePageForceUpdateTest),
```

Comment:
This is not a standalone test anymore. We decided everything that uses mocks should be in  `mock: []`

* Kompakt mocks
* Offline mocking
* Force Update / Soft Update MC mocking